### ISSUE TITLE
Add activity to Carp lake

### ISSUE THREAD

**Issue by JLWade** [bcgov/bcparks-ar-api#324]
#### Description:

Add **frontcountry** cabins activity to A&R for the Islands sub-area of Carp Lake

![Image](https://github.com/bcgov/bcparks-ar-api/assets/38668130/7be72f3f-ddfd-492d-a29a-f9f4e9e71f03)

#### Acceptance Criteria:
- "**Frontcountry** Cabins" activity is added to the enter data page under Carp Lake Park, sub-area Islands
- Activity added to exported report

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by cameronpettit** [comment]
@JLWade Frontcountry or Backcountry Cabins?

**Comment by cameronpettit** [comment]
Completed in Prod only.

**Comment by manuji** [comment]
Tested on PROD: **Passed**
* Verified that the Frontcountry Cabins activity has been added to the Carp lake park, Island sub area from the DynamoDB and it is displayed on the A&R PROD site. 
* Verified that the activity is displayed on the exported report as well.
* Passing to PO review. 


Note: I made a change on PROD environment for testing purposes to verify that the new activity is included to the exported report and then reverted that to empty.
